components/scream/src/dynamics/homme/dynamics_driven_grids_manager.cpp

```diff
@@ -107,10 +112,7 @@ build_grids (const std::set<std::string>& grid_names,
       pg_codes.push_back(code);
     }
   }
-  // Also check the ref grid (and retrieve its code)
-  EKAT_REQUIRE_MSG (supported_grids().count(reference_grid)==1,
-                    "Error! Grid '" +reference_grid + "' is not supported by this grid manager.\n");
-  pg_codes.push_back(m_grid_codes.at(reference_grid));
+  pg_codes.push_back(m_grid_codes.at(m_ref_grid_name));
```

Comment:
This is really just a general naming convention comment.  I foresee some confusion in the future when we bring in the PhysGrid stuff, since this is typically referred to as "PG".  So we might want to start thinking about how we thing that grid will be names within the context of scream.

Reply:
I think PG should refer to any physics grid, and what is current called "PG" should probably be called FV phys grid (the Finite Volume physics grid), as opposed to the GLL phys grid.

I understand that people are used to call the FV grid just "PG", by now, and "names battles" are a waste of energy. But fwiw, I would argue PG should just mean "physics grid". We can then decide that pgN means FV grid for N>0 and GLL grid for N=0 (like it's done in the code).

Reply:
I'm in agreement on the naming.  I'm not sure we can win the names battle, but we do have some agency at the ground level.  In a few months, when we begin work on implementing the FV grid we can lean into a naming convention there that makes more sense.  I just mention it now so that it is on the radar.

Reply:
In homme, the fv grid is already called fv. The module is `gllvfremap`, underscoring that the two grids are GLL and FV. Imho, we can simply keep the naming convention used in Homme also in scream.